src/libraries/Microsoft.Extensions.Caching.Abstractions/src/CompatibilitySuppressions.xml

```diff
@@ -0,0 +1,32 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- https://learn.microsoft.com/en-us/dotnet/fundamentals/package-validation/diagnostic-ids -->
+<Suppressions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <Suppression>
+    <DiagnosticId>CP0002</DiagnosticId>
+    <Target>M:Microsoft.Extensions.Caching.Memory.IMemoryCache.GetCurrentStatistics</Target>
+    <Left>lib/net6.0/Microsoft.Extensions.Caching.Abstractions.dll</Left>
+    <Right>lib/netstandard2.0/Microsoft.Extensions.Caching.Abstractions.dll</Right>
+    <IsBaselineSuppression>true</IsBaselineSuppression>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0002</DiagnosticId>
+    <Target>M:Microsoft.Extensions.Caching.Memory.IMemoryCache.GetCurrentStatistics</Target>
+    <Left>lib/net7.0/Microsoft.Extensions.Caching.Abstractions.dll</Left>
+    <Right>lib/netstandard2.0/Microsoft.Extensions.Caching.Abstractions.dll</Right>
+    <IsBaselineSuppression>true</IsBaselineSuppression>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP1002</DiagnosticId>
+    <Target>System.Runtime, Version=4.1.2.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a</Target>
+    <Left>lib/net6.0/Microsoft.Extensions.Caching.Abstractions.dll</Left>
+    <Right>right</Right>
+    <IsBaselineSuppression>true</IsBaselineSuppression>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP1002</DiagnosticId>
+    <Target>System.Runtime, Version=4.1.2.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a</Target>
+    <Left>lib/net7.0/Microsoft.Extensions.Caching.Abstractions.dll</Left>
+    <Right>right</Right>
+    <IsBaselineSuppression>true</IsBaselineSuppression>
+  </Suppression>
```

Comment:
We need to understand these suppressions. I'm not sure which assembly reference this refers to.